### PR TITLE
Configure listening time out 

### PR DESCRIPTION
**Pull request by alexis-kochetov**
image
image

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/ListeningSettingsModal.razor [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+@{
+    var m = State.Value;
+    var listeningMode = m.ListeningMode;
+    var defaultTimeout = (int)Math.Floor(AudioSettings.IdleListeningTimeout.TotalMinutes);
+}
+<DialogFrame Title="Listening settings" Class="transcription-options-modal modal-sm" HasCloseButton="false">
+    <Body>
+    <FormBlock>
+        <Tile>
+        <TileItem Click="() => UpdateListeningMode(ListeningMode.Default)">
```
Comment (frolyo) [inline src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/ListeningSettingsModal.razor:15]: Candidate for template func

File: src/dotnet/Core/Collections/ArrayExt.cs [hunk 2]
```diff
@@ -92,4 +92,13 @@ public static void Deconstruct<T>(this T[] array, out T? first, out T? second, o
 
     public static void Deconstruct<T>(this T[] array, out T? first, out T? second, out T? third, out T? fourth, out T? fifth, out T[] rest)
         => (first, second, third, fourth, (fifth, rest)) = array;
+
+    public static ApiArray<T> ToApiArray<T>(this T[] array)
```
Comment (frolyo) [inline src/dotnet/Core/Collections/ArrayExt.cs:96]: don't we already have extension?

Comment (frolyo) [reply src/dotnet/Core/Collections/ArrayExt.cs:96]: image

**Review by frolyo — APPROVED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/ListeningSettingsModal.razor (hunk 1 above)
Comment (alexis-kochetov) [reply src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/ListeningSettingsModal.razor:15]: I don't get you, please provide some context or an example

**Pull request merged**